src/components/portal/portal.tsx

```diff
@@ -0,0 +1,119 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+
+@Component({
+    tag: 'limel-portal',
+    shadow: true,
+    styleUrl: 'portal.scss',
+})
+export class Portal {
+    @Prop()
+    public visible = false;
+
+    @Prop()
+    public containerId: string;
```

Comment:
It would be nice to have some documentation for this component, and maybe for the menu-surface component too. I think we can add docblocs without the components automatically showing up in docz, right? If not, maybe we can just use `//` style comments?

Reply:
Yeah it should be possible! I think they should only show up if we have an `.mdx`-file for it